app/src/components/ui/DataTable/Row.tsx

```diff
@@ -0,0 +1,56 @@
+import { flexRender, Row as IRow } from "@tanstack/react-table";
+import { useCallback } from "react";
+
+import { TableCell, TableRow } from "@/components/ui/table";
+
+const NUMBER_STYLE = {
+  fontFamily: "sans-serif",
+  fontVariantNumeric: "tabular-nums",
+};
```

Comment:
I think the only numbers we're showing always are the monetary values. Maybe this should be tailored specifically for that?

Reply:
This formatting is intended both for pure numbers as well as currency. The masking of currency should be handled in the `columns` definition used for creating the data table.